<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LobbyBrowser</class>
 <widget class="QMainWindow" name="LobbyBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LobbyBrowser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="serverAddressEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="serverConnectButton">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QListWidget" name="lobbyList"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="newLobbyNameEditor"/>
            </item>
            <item>
             <widget class="QPushButton" name="createLobbyButton">
              <property name="text">
               <string>Create Lobby</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="lobbyName">
            <property name="text">
             <string>No Lobby Selected</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="refreshButton">
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="joinButton">
            <property name="text">
             <string>Join</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="leaveButton">
            <property name="text">
             <string>Leave</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="readyButton">
            <property name="text">
             <string>Ready</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="notReady">
            <property name="text">
             <string>Not Ready</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="launchButton">
            <property name="text">
             <string>Launch</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>853</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
